projects/module-1/ideabox-group.md

```diff
@@ -107,14 +109,14 @@ As a user,
 
 As a user,
 - When I click the "Delete" button on an idea card,
-- The card instance should be permanently removed from the ideas array
-- The card should be permanently removed from my view
+- That idea object should be permanently removed from the ideas array
+- The idea card should be permanently removed from the DOM
```

Comment:
I like that this verbiage helps clarify that the card shouldn't just be hidden - it should be removed from the array then the updated array should be re-rendered on the DOM